roles/openshift_health_checker/callback_plugins/zz_failure_summary.py

```diff
@@ -59,6 +59,28 @@ def _print_failure_summary(self):
                 indented = u'{}{}'.format(subsequent_indent, entry)
                 self._display.display(indented)
 
+        if self.__failures:
+            failed_checks = [
+                name
+                for name, result in failure['result']._result.get('checks', []).items()
+                for failure in self.__failures
+                if result.get('failed', False)
+            ]
+            # FIXME: get name of currently running playbook, if possible.
+            NAME_OF_PLAYBOOK = 'playbooks/byo/config.yml'
+            msg = (
+                "\nThe execution of the playbook '{}' includes checks designed "
+                'to ensure it can complete successfully. One or more of these '
```

Comment:
Actually this message is directed at the checks being run as part of the install playbook. I would like it to be different for running the pre-install playbook explicitly, and for running health checks. I think it would make sense to add a role variable that the playbook sets to give some context for which of those it is, so this plugin can adjust output accordingly.

Also if I'm not wrong, this gets printed for every host, which seems excessive. I'll try to address these things.